### DISCUSSION TITLE
Can the um's be left in?

### DISCUSSION THREAD

**Discussion by richpav** [openai/whisper#286] [Q&A] [answered]
Whisper does a great job removing filler words and stutters. Can that be disabled so the transcript reflects exactly what was spoken?

**Comment by jianfch** [comment, marked as answer]
The heuristics do not suppress disfluencies by default. It is likely the result of training on data that, presumably, has little to no disfluencies in its transcripts.
Disfluencies are in the vocab, so you can try to give it a `prompt` with the disfluencies you want it to predict.

**Comment by LaurinmyReha** [comment]
This should help you regarding transcribing in a more verbatim fashion.
https://github.com/nyrahealth/CrisperWhisper

Further this gives a more theoretical insight into how transcription policy influences training
https://arxiv.org/pdf/2607.18934
